AppController/lib/hermes_service.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/ruby -w
+
+$:.unshift File.join(File.dirname(__FILE__))
+require 'helperfunctions'
+require 'monit_interface'
+
+# Starts and stops the Hermes (messenger) service.
+module HermesService
+
+  # Starts the Hermes service on this machine. We don't want to monitor
+  # it ourselves, so just tell monit to start it and watch it.
+  def self.start()
+    start_cmd = "/usr/bin/python /root/appscale/Hermes/hermes.py"
+    # The stop command doesn't work and relies on terminate.rb.
+    stop_cmd = "/usr/bin/pkill -9 hermes_service"
+    MonitInterface.start(:hermes, start_cmd, stop_cmd, "9999", {})
```

Comment:
Yeah I've been wondering about the rest of them that have 9999 appended. I think we should drop it altogether.